.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ _Please describe your pull request_
 
 _Please go through this checklist and make sure all applicable tasks have been done_
 
-- [ ] Update/write design documentation in `./design`
+- [ ] Update/write design documentation in `./development-docs/design`
```

Comment:
I'm not sure this should stay there. I think nobody ever did that in the last year or how long it is there. Maybe you can remove it?

Reply:
Yeah it is a weird one, because by the time you get to a PR the design doc is probably redundant. I will remove it.